### PR TITLE
New KubeVirt Live Migration Lab

### PR DESCRIPTION
**Pull request by cwilkers**
- Added new lab to toc
- Added new quickstart include for multinode minikube
- Added example vm with masquerade network
- Added scriptlets for:
  - live migration feature gate
  - port exposure
  - netcat http server while loop
  - tests for http

Signed-off-by: cwilkers <cwilkers@redhat.com>

### CODE REVIEW

**Comment by cwilkers** [comment]
Following a conversation on the kubevirt-dev mailing list, I am considering including labs/manifests/vm.yaml in this PR, changing its network style to masquerade, and re-using this yaml for this PR rather than duplicating it in a code scriptlet.

File: labs/kubernetes/migration.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+kubelet: Running
+```
+
+Check that kubevirt fully deployed:
+
+```bash
+kubectl -n kubevirt get kubevirt
+NAME       AGE     PHASE
+kubevirt   3m20s   Deployed
+```
```
Comment (mazzystr) [inline labs/kubernetes/migration.md:65]: L25-65 should be removed.  Provider information isn't to the lab.  I could compromise with an annotation stating Live Migration reqs two or more nodes and a properly configured sdn.

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by mazzystr** [comment]
Everything looks great!  We're rolling!

/lgtm
/approve

**Comment by cwilkers** [comment]
Fixed #759

**Comment by cwilkers** [comment]
Fixed #760

**Pull request merged**